lib/shopify_cli/theme/dev_server/hot-reload-tae.js

```diff
@@ -0,0 +1,167 @@
+
+(() => {
+  const APP_BLOCK = 'app-block', APP_EMBED_BLOCK = 'app-embed-block';
```

Comment:
@karreiro I might make these bools or integers to avoid costly string comparisons

Reply:
I'm not sure if it would have a high impact in the DX (generally, I would focus on DOM calls for optimizations, as they are the most expensive ones).